tests/python/builtin.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+import unittest
+import subprocess
+
+
+class TestBuiltin(unittest.TestCase):
+
+    def run_test(self, cmd, expected):
+        p = subprocess.Popen(
+            [cmd], shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        self.assertRegexpMatches(p.stdout.readline(), expected)
+
+    def test_pid(self):
+        cmd = """test=pid; ../.././build-release/src/bpftrace -e \
+		'i:ms:1 { printf(\"SUCCESS '$test' %d\\n\", pid); exit(); }' \
+		 | grep '^SUCCESS '$test' [0-9][0-9]*$' || echo "FAILURE $test"
+		"""
+        self.run_test(cmd, "^(SUCCESS pid) [0-9][0-9]*$")
```

Comment:
Can you consolidate this strings with the one in the grep above into a single variable please? It would just help avoid mistakes